posthog/schema.py

```diff
@@ -3583,6 +3583,7 @@ class WebOverviewQueryResponse(BaseModel):
     timings: Optional[list[QueryTiming]] = Field(
         default=None, description="Measured timings for different parts of the query generation process"
     )
+    usedPreAggregatedTables: Optional[bool] = None
```

Comment:
What do we use this for? Just metrics? Also, why make it optional? Just assign `False` as the default